app/plugins/Wallet/js/lifecycle.js

```diff
@@ -26,12 +26,14 @@ addResultListener('update-status', function(result) {
 	// Update balance confirmed and uncomfirmed
 	var bal = convertSiacoin(wallet.confirmedsiacoinbalance);
 	var pend = convertSiacoin(wallet.unconfirmedincomingsiacoins).sub(convertSiacoin(wallet.unconfirmedoutgoingsiacoins));
-	if (wallet.Unlocked && wallet.Encrypted) {
+	if (wallet.unlocked && wallet.encrypted) {
 		eID('confirmed').innerHTML = 'Balance: ' + bal + ' S';
 		eID('uncomfirmed').innerHTML = 'Pending: ' + pend + ' S';
+		eID('confirmed').style.removeProperty('display');
+		eID('uncomfirmed').style.removeProperty('display');
```

Comment:
These fields are now only displayed when the wallet is unlocked. If the wallet is locked (or has not been created), we set `display:none` on those elements. When the wallet is unlocked, we remove the `display` property so that they inherit the property from their parent css.